frontend/apps/erd-web/app/erd/p/[...slug]/page.tsx

```diff
@@ -23,8 +25,10 @@ export default async function Page({
 
   const input = await res.text()
 
-  // Currently supports Postgres only
-  const { value: dbStructure, errors } = await parse(input, 'postgres')
+  setPrismWasmUrl(path.resolve(process.cwd(), 'prism.wasm'))
```

Comment:
process.cwd() gives the root path of Next.js.
By following the file from here, the wasm file can be accessed in the local environment as well as on Vercel.

ref: [Reading files on Vercel during Next.js ISR | François Best](https://francoisbest.com/posts/2023/reading-files-on-vercel-during-nextjs-isr)